internal/integration/change_log_test.go

```diff
@@ -60,18 +60,18 @@ func (suite *ChangelogSuite) TestGetAllChangelogsHandler() {
 // Uncomment after changelog-CR is merged
 //
 func (suite *ChangelogSuite) TestGetSingleChangelogHandler() {
-	resp, err := http.Get("http://localhost:4000/change_logs/400")
+	resp, err := http.Get("http://localhost:4000/change_logs/400/?limit=999&offset=0")
```

Comment:
we should add a test that with a shorter limit and an offset greater than 0, and verify the correct subset of the changelog is returned.